go.mod

```diff
@@ -165,6 +164,7 @@ replace github.com/emicklei/go-restful => github.com/emicklei/go-restful/v3 v3.8
 
 replace golang.org/x/crypto => golang.org/x/crypto v0.1.0
 
-// Containerd package 1.6.x has a dependency of containernetworking/cni > 1.0.0 and which also has a CNI change spec 1.0.0
-// To keep it backward compatible to the spec 0.4.0, we will pin it to v1.5.x
-replace github.com/containerd/containerd => github.com/containerd/containerd v1.5.18
+// Containernetworking packages must remain pinned until VPC CNI moves to CNI spec 1.0.0
+replace github.com/containernetworking/cni => github.com/containernetworking/cni v0.8.1
```

Comment:
why we update `github.com/containernetworking/cni v1.1.1`

Reply:
The callout for k8s 1.26 is to use a containerd version >= 1.6.0. Containerd is an indirect dependency for us, but it moving to v1.6.6 moves `containernetworking` packages past 1.0.0. We do not want `containernetworking` to move past 1.0.0, as that forces us to use CNI spec 1.0.0, which we cannot support on older EKS versions. Trying to use `types/040` from `containernetworking/cni` does not fully work as other packages use `types/100` and there is no way to cast forward